gossip/discovery/discovery_impl.go

```diff
@@ -735,9 +735,9 @@ func (d *gossipDiscoveryImpl) expireDeadMembers(dead []common.PKIidType) {
 
 	d.lock.Unlock()
 
-	for _, member2Expire := range deadMembers2Expire {
-		d.logger.Warning("Closing connection to", member2Expire)
-		d.comm.CloseConn(&member2Expire)
+	for i := range deadMembers2Expire {
+		d.logger.Warning("Closing connection to", deadMembers2Expire[i])
+		d.comm.CloseConn(&deadMembers2Expire[i])
```

Comment:
I don't see the benefit of changing the code to look up the value twice by index instead of just directly using the value provided by for-range, as the code did before

Reply:
I don't see how this is related to the pull request's intention.

Reply:
Do not consider that. This is the result of a forced push that I have done to sign my commits. I accidentally signed commits that were not mine. I will fix that and probably force-push again.

Reply:
no worries